### PR TITLE
156 add reaction to calendar

### LINKED ISSUE #156: Add reaction to calendar
**Short description:**
Add create new calendar reaction, add getter url with all available calendars

**Describe the solution you'd like**
I want a reaction wich can create a new calendar
I want a route that return all the calendars id to the front

### PR DESCRIPTION
**Pull request by GuyomT**
# Purpose
add the create calendar reaction to the service Google Calendar
Fixes #156

## Added features

* reaction: create calendar
* route getAvailableCalendars

## Updated features

* createEvent now take the calendarID in parameter and it's working

## Removed features

* removed create event from the bot Area google account cause useless

### CODE REVIEW

File: server/server_app.js [hunk 1]
```diff
@@ -801,8 +802,31 @@ app.get(
 )
 
 /**
- * List available performers, such as bot/user.
- * Route protected by a JWT token
+ * @brief List all available calendars where the user is.
+ */
```
Comment (LucasTesnier) [inline server/server_app.js:808]: Add 'Route protected by...'

File: server/services/calendar/reactions/create_calendar.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      return response.data
+    }
+  } catch (err) {
+    console.log(`An error occurred in create calendar: ${err}`)
+    throw err
+  }
+}
+
+/**
+ *
```
Comment (LucasTesnier) [inline server/services/calendar/reactions/create_calendar.js:64]: desc missing

**Review by LucasTesnier — CHANGES_REQUESTED** [review]
doc

File: server/docs/index.html [hunk 3]
```diff
@@ -1,13 +0,0 @@
-<!doctype html>
```
Comment (ZQUEMA) [inline server/docs/index.html]: Why is this file removed ?

**Review by ZQUEMA — APPROVED** [review]

**Review by LucasTesnier — APPROVED** [review]
